common/src/main/java/com/microsoft/identity/common/internal/logging/LoggerSettings.java

```diff
@@ -0,0 +1,43 @@
+package com.microsoft.identity.common.internal.logging;
+
+public class LoggerSettings {

```

Comment:
Comments are always welcomed! Just not sure how are we going to organize all the other settings  in the future.

Reply:
I redesigned the settings for logger, where I move the PII and logcat settings back into the Logger class and remove the LoggerSettings class. I think this will be the final change until we have a better unified strategy on the settings. Thanks.